rules-reviewed/technology-usage/javaee-technology-usage.windup.xml

```diff
@@ -58,41 +58,41 @@
                 </or>
             </when>
             <perform>
-                <technology-tag level="INFORMATIONAL">Java EE CDI</technology-tag>
+                <technology-tag level="INFORMATIONAL">CDI</technology-tag>
             </perform>
         </rule>
         <rule id="javaee-technology-usage-00021">
             <when>
                 <graph-query discriminator="TechnologyTagModel">
-                    <property name="name">Java EE CDI</property>
+                    <property name="name">CDI</property>
                 </graph-query>
             </when>
             <perform>
-                <technology-identified name="Java EE CDI">
-                    <tag name="CDI"/>
+                <technology-identified name="CDI">
+                    <tag name="Execute"/>
                     <tag name="Processing"/>
                     <tag name="Java EE"/>
                 </technology-identified>
             </perform>
         </rule>
-        <!-- JSON-P identication -->
+        <!-- JSON-P identification -->
         <rule id="javaee-technology-usage-00030">
             <when>
                 <javaclass references="javax.json.{*}"/>
             </when>
             <perform>
-                <technology-tag level="INFORMATIONAL">Java EE JSON-P</technology-tag>
+                <technology-tag level="INFORMATIONAL">JSON-P</technology-tag>
```

Comment:
I think it is important to keep the "Java EE" prefix (or something like it). "JSON-P" is an overloaded acronym, and the more common usage isn't the one referenced here.